src/molgenis/capice_resources/train_data_creator/data_parsers/clinvar.py

```diff
@@ -54,6 +54,10 @@ def _obtain_gene(self, clinvar_frame: pd.DataFrame) -> None:
                 Performed inplace.
         """
         self._process_column(clinvar_frame, TrainDataCreatorEnums.GENE.value, 'GENEINFO=')
+        # For now, deleting the SYMBOL ID as VKGL does not yet export this
```

Comment:
FYI We wrote a tool to produce better VKGL output: https://github.com/molgenis/vkgl-consensus which is not used at the moment pending https://trac.molgeniscloud.org/ticket/8633. The output of this tool contains gene symbols and would be a good candidate to extend with gene identifiers.

Reply:
So we can either make a new ticket to include this new VKGL output after this PR is merged, or this PR is blocked on that story. 
Taking into account both the severity of this bug I would suggest merging this PR, creating an issue that we don't do anything with SYMBOL ID's and another issue that we don't support multiple SYMBOL ID's per single variant, creating a new ticket to address these issues.